test/output-tests.js

```diff
@@ -27,7 +27,10 @@ Object.keys(files).forEach((absPath) => {
 
 describe("output", () => {
   function check(data) {
-    const code = compile(data.actual).code;
+    const code = compile(data.actual, {
+      parse: require("../lib/parsers/acorn.js").parse
```

Comment:
Do you want to force all parser passes to use this or should we add in skip logic similar to other areas.